apps/nestjs-api/src/reminder-emails/reminder-emails.service.ts

```diff
@@ -730,25 +710,21 @@ export class ReminderEmailsService {
       .replace(/\${menteeFirstName}/g, `${match.menteeFirstName}`)
 
     const params = {
-      Destination: {
-        ToAddresses: isProductionOrDemonstration()
-          ? [match.mentorEmail]
-          : [this.config.get('NX_DEV_MODE_EMAIL_RECIPIENT')],
-        BccAddresses: [SENDER_EMAIL],
-      },
-      Message: {
-        Body: {
-          Html: { Data: sanitizedHtml },
-        },
-        Subject: { Data: sanitizedSubject },
-      },
-      Source: `${SENDER_NAME} <${SENDER_EMAIL}>`,
+      from: `${SENDER_NAME} <${SENDER_EMAIL}>`,
+      to: isProductionOrDemonstration()
+        ? match.mentorEmail
+        : this.config.get('NX_DEV_MODE_EMAIL_RECIPIENT'),
+      bcc: SENDER_EMAIL,
+      subject: sanitizedSubject,
+      html: sanitizedHtml,
     }
 
-    this.ses.sendEmail(params, (err, data) => {
-      if (err) console.log(err, err.stack)
-      else console.log(data)
-    })
+    try {
+      await this.transporter.sendMail(params)
+      console.log('Email sent successfully')
+    } catch (err) {
+      console.error('Error sending email:', err)
+    }
```

Comment:
The `params` and the `try ... catch` code repeat in multiple places. It would be nice to extract this logic into a helper method at some point.